cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1599,7 +1599,7 @@ msgid "Tell us more about your problem"
 msgstr "Dywedwch fwy wrthym am eich problem"
 
 #: cla_public/apps/contact/forms.py:150
-msgid "Your notes must be 4000 characters or less"
+msgid "Your notes must be 4000 characters or fewer"
 msgstr "Rhaid i’ch nodiadau fod yn 4000 nod neu lai"
```

Comment:
Minor change here, but do we want to change the last word in Welsh to fewer or can we assume it means the same thing?

Reply:
GDS do not prescribe Welsh wording, so we leave it as it is.  It is just an improvement in the quality of the English, we have no reason to believe that the Welsh is of poor quality too, nor do we know whether the same grammar rules vis-à-vis countable and non-countable items apply.